com.unity.ml-agents/Runtime/DiscreteActionMasker.cs

```diff
@@ -6,12 +6,15 @@
 namespace MLAgents
 {
     /// <summary>
+    /// The DiscreteActionMasker class represents a set of masked (disallowed) actions and
+    /// provides utilities for setting and retrieving them.
+    /// </summary>
+    /// <remarks>
     /// Agents that take discrete actions can explicitly indicate that specific actions
     /// are not allowed at a point in time. This enables the agent to indicate that some actions
     /// may be illegal (e.g. the King in Chess taking a move to the left if it is already in the
-    /// left side of the board). This class represents the set of masked actions and provides
-    /// the utilities for setting and retrieving them.
-    /// </summary>
+    /// left side of the board). 
```

Comment:
@vincentpierre Other chess references were already here.

Reply:
Well, we can fix things that are wrong in the original text, too.